reader-gtfs/src/main/java/com/conveyal/gtfs/model/Entity.java

```diff
@@ -100,7 +100,7 @@ protected boolean checkRangeInclusive(double min, double max, double actual) {
         private String getFieldCheckRequired(String column, boolean required) throws IOException {
             String str = reader.get(column);
             if (str == null) {
-                if (!missingRequiredColumns.contains(column)) {
+                if (required && !missingRequiredColumns.contains(column)) {
```

Comment:
There's one difference between both implementations. The `reader.get(..)` method in `javacsv` returns an empty string if the column is not existing, whereas in `opencsv` it returns `null`. This now leads to this `MissingColumnError` for required fields instead of an `EmptyFieldError`. I also added the `required` check here, as otherwise the `MissingColumError` was raised even for required fields.